source/Irrlicht/COpenGLDriver.cpp

```diff
@@ -989,28 +973,22 @@ bool COpenGLDriver::genericDriverInit()
 	// Reset The Current Viewport
 	glViewport(0, 0, Params.WindowSize.Width, Params.WindowSize.Height);
 
-	glEnable(GL_FRAMEBUFFER_SRGB);
-    glDisable(GL_DITHER);
-    glDisable(GL_MULTISAMPLE);
-    glEnable(GL_TEXTURE_CUBE_MAP_SEAMLESS);
-    extGlClipControl(GL_UPPER_LEFT, GL_ZERO_TO_ONE);
-	glClearDepth(0.0);
-	glDepthFunc(GL_GEQUAL);
-	glDepthRange(1.0,0.0);
-	glFrontFace(GL_CCW);
-
 	// adjust flat coloring scheme to DirectX version
 	///extGlProvokingVertex(GL_FIRST_VERTEX_CONVENTION_EXT);
 
-	// create material renderers
-	createMaterialRenderers();
-
-	// set the renderstates
-	setRenderStates3DMode();
-
 	// We need to reset once more at the beginning of the first rendering.
 	// This fixes problems with intermediate changes to the material during texture load.
-	ResetRenderStates = true;
+    SAuxContext* found = getThreadContext_helper(false);
+    glEnable(GL_FRAMEBUFFER_SRGB);
+    glDisable(GL_DITHER);
```

Comment:
enable

Reply:
you sure? as for now it's been disabled https://github.com/Crisspl/IrrlichtBAW/blob/master/source/Irrlicht/COpenGLDriver.cpp#L993

Reply:
yes I'm sure, this stuff could help with 6bit (g-sync) displays

Reply:
resolved